wordpress-mobile/WordPress-Android
# Add tracking for whitelisted ips settings

Adds tracking to the jetpack whitelisted ips security settings.
- jetpack_whitelisted_ips_viewed when the list of ips is viewed
- jetpack_whitelisted_ips_changed when the user updates the settings (it's tracked when we store the updated settings into a persistent storage = when the user leaves the screen)

To test:
1. My Site
2. Settings
3. Security
4. Open Whitelisted IP addresses
5. Notice "jetpack_whitelisted_ips_viewed" is tracked
6. Press back and notice nothing is tracked
7. Open Whitelisted IP addresses
8. Add an address eg "12.12.12.1"
9. Press back and notice "jetpack_whitelisted_ips_changed" is tracked

PR submission checklist:
